src/_routing/_components/ProfileNavigator.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { FC } from 'react';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+
+import { theme } from '../../_styles/theme';
+import History from '../../history/History';
+import Profile from '../../profile/Profile';
+
+export type TProfileParams = {
+  History: undefined;
+  Profiel: undefined;
+};
```

Comment:
This should be in line with the other navigators: 

```suggestion
export type TProfileRoutes = 'Profile' | 'History';
export type TProfileParams = Record<TProfileRoutes, undefined>;
```

Furthermore, there's a dutch word in there 🚨

Reply:
opgelost in volgende pr